.github/workflows/android_tests.yml

```diff
@@ -39,5 +41,28 @@ jobs:
           architecture: x64
       - run: ./ci/mac_ci_setup.sh
         name: 'Install dependencies'
-      - run: bazel test --test_output=all --build_tests_only //test/java/...
+      - run: |
+          # /test/kotlin/integration is excluded since it is run on a separate job
+          bazelisk query //test/java/...
         name: 'Run Java library tests'
+  kotlintestslinux:
+    name: kotlin_tests_linux
+    runs-on: ubuntu-18.04
+    timeout-minutes: 90
+    container:
+      image: envoyproxy/envoy-build-ubuntu:e33c93e6d79804bf95ff80426d10bdcc9096c785
+      env:
+        CC: /opt/llvm/bin/clang
+        CXX: /opt/llvm/bin/clang++
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '8'
+          java-package: jdk
+          architecture: x64
+      - run: |
+          bazel test --test_output=all --build_tests_only //test/kotlin/...
+        name: 'Run Kotlin library integration tests'
```

Comment:
Add newline.

Also maybe add brief comment about why java_tests_linux is disabled?